design-library/src/components/BccInfo/BccInfo.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { CheckCircleIcon } from "@bcc-code/icons-vue";
+import { describe, it, expect } from "vitest";
+
+import { mount } from "@vue/test-utils";
+import BccInfo from "./BccInfo.vue";
+
+describe("BccInfo", () => {
+  it("renders title", async () => {
+    const wrapper = mount(BccInfo, { props: { title: "Title" } });
+    expect(wrapper.html()).toContain("Title</h4>");
+    expect(wrapper.html()).not.toContain("h-4 w-4");
+  });
+
+  it("renders icon and text", async () => {
+    const wrapper = mount(BccInfo, {
+      props: { title: "Title", infoIcon: CheckCircleIcon, infoText: "Info text" },
+    });
+    expect(wrapper.html()).toContain("Title</h4>");
+    expect(wrapper.html()).toContain("h-4 w-4");
```

Comment:
Instead of testing these classes here (that should be moved to a named class anyways), instead you can test if the component actually renders, which is much more valuable for a unit test:

```ts
        expect(wrapper.findComponent(CheckCircleIcon).exists()).toBe(true);
```